### DISCUSSION TITLE
Adding Google Maps API

### DISCUSSION THREAD

**Discussion by boehmap** [GUI/covid-vaccine-finder#29] [Ideas]
Hey team,
This resource is amazing. I'm digging through the source code, and I think I can help contribute by adding a Google Map option to the page. 
I'm a little less familiar with the tech stack here. The two "I don't know how this works" things that come to mind are to scrape the titles of the stores and pull out the address. The other is to add items to a javascript object as each store is displayed, in line, and then load those items into the map.
The best (but falls under the "If I knew what I was doing" category) would be to just have a map on a separate page, and use the existing store/appointment templates to fill out populate the map with addresses, and info boxes with the appointment information.
If anyone is comfortable with me taking a swing at any of these, please let me know. I'll start playing around.

**Comment by GUI** [comment]
@boehmap: Thanks so much for the offer! I've actually been e-mailing with someone that has been working on map, so I think we might have a pull request coming in sometime soon-ish. So we don't duplicate efforts, I think it might make most sense to wait on that, and then we can maybe collaborate on that pull request. I'll be sure to provide a link to the PR once it's there. Thank you, though!

**Reply by boehmap** [reply]
Thought that might be the case. That's why I threw a note here. Happy to take a look at the PR

**Reply by GUI** [reply]
Maps are now implemented. There's some further discussion in https://github.com/GUI/covid-vaccine-spotter/pull/35, but sorry I forgot to update this thread.
